### ISSUE TITLE
Editor plugins in WYSIWYG mode

### ISSUE THREAD

**Issue by mvanassche** [nhn/tui.editor#1214]
Allow editor plugin (in WYSIWYG mode).
Plugins now are mostly oriented towards view mode.

It would be useful to allow a plugin in the WYSIWYG mode to be an editor.
Useful examples are #1158 , a monaco-based code editor, a visual UML editor, ...

**Comment by seonim-ryu** [comment]
@mvanassche That's right. Currently, when using plugins such as UML and chart in the WYSIWYG editor, many people feel that it is not convenient because editing through popup is required. As you said, there should be a UI that allows you to check the plugin's rendering result right inside the WYSIWYG editor. We are aware of this issue, and we will be working on improving the editor after the 3.0 major update.
